.clj-kondo/config.edn

```diff
@@ -794,7 +794,7 @@
    metabase-enterprise.serialization.test-util/with-world                       macros.metabase-enterprise.serialization.test-util/with-world
    metabase-enterprise.test/with-gtaps!                                         macros.metabase-enterprise.sandbox.test-util/with-gtaps!
    metabase-enterprise.advanced-permissions.api.util-test/with-impersonations!  macros.metabase-enterprise.advanced-permissions.api.util-test/with-impersonations!
-   metabase-enterprise.query-field-validation.api-test/with-test-setup          macros.metabase-enterprise.query-field-validation.api-test/with-test-setup
+   metabase-enterprise.query-field-validation.api-test/with-test-setup!         macros.metabase-enterprise.query-field-validation.api-test/with-test-setup!
```

Comment:
Can roll this back. Kondo made me do it for an earlier version where I was calling bang method in the implementation.

Reply:
Rolled it back